### ISSUE TITLE
Suggestion: Macaw's mods compatibility

### ISSUE THREAD

**Issue by Miszelka** [Snownee/SnowRealMagic#301]
### Mod loader

Forge

### Minecraft version

1.18.2

### Mod version

4.2.4

### Modloader version

forge-40.2.0

### If bug:

- [ ] Can you reproduce this issue with relevant mods only?

### Issue description

It would be nice if Macaw's Roofs changed textures when snowed on (like stairs), Macaw's Fences and Walls do, but the textures can be wonky (so I'm not sure which creator to contact or if this is even possible).

**Comment by kerryghan-relot** [comment]
Indeed, I noticed a bug where diagonal connected fences disconnect when covered in snow.

**Comment by vinnyg0621** [comment]
yeah I'd love to see the snowy roofs too.
